Created a new entry for numpy.aMax (#2463)
Codecademy/docs: content/numpy/concepts/built-in-functions/terms/amax/amax.md

The **`.amax()`** function returns the maximum value of an array or maximum along an axis. The `.amax()` function is equivalent to the NumPy method `ndarray.max()`.

## Syntax

```pseudo
numpy.amax(a, axis, out, keepdims, initial, where)
```

The `a` parameter is required and represents the array of elements to choose the maximum from. All other parameters are optional.

Parameters of the `.amax()` function:

- `a`: (Input) The array of elements to choose the maximum from.
- `axis`: (Default = None) An int or a tuple of ints specifying the axis/axes along which to operate. If a tuple of ints, the maximum is across multiple axes.
- `out`: (Default = None) An [`ndarray`](https://www.codecademy.com/resources/docs/numpy/ndarray) to receive result. Must have the same shape as the expected output.
- `keepdims`: (Default = <no value>) A boolean; if `True` will keep reduced axes in the result as dimensions with size one.
- `inital`: (Default = <no value>) The minimum value of an output element.
- `where`: (Default = <no value>) A boolean array that maps to `a` with identical dimensions and selects which elements to compare for the maximum.

Returns:
The maximum element in the array `a`.

> - If axis is None, the result would be a scalar value.

Reviewer: ```suggestion
- If `axis` is None, the result would be a scalar value.

```

> - If axis is an integer, the result would be an array of dimension `a.ndim - 1`.

Reviewer: ```suggestion
- If `axis` is an integer, the result would be an array of dimension `a.ndim - 1`.

```

> - If axis is a tuple, the result would be an array of dimension `a.ndim - len(axis)`.

Reviewer: ```suggestion
- If `axis` is a tuple, the result would be an array of dimension `a.ndim - len(axis)`.

```

## Example

The following example creates an array and then uses a few `.amin()` operations to return the minimum.

```py
import numpy as np

nd = np.array([[2,4,6,8],[10,12,14,16]])

print(np.amax(nd))
print(np.amax(nd, axis=0))
print(np.amax(nd, axis=1))
```

This produces the following output:

```shell
16

> [10,12,14,16]

Reviewer: ```suggestion
[10 12 14 16]
```

---
Title: '.amax()'

> Description: 'Returns the maximum of an array or maximum along an axis.'

Reviewer 2: ```suggestion
Description: 'Returns the maximum of a given array or maximum along an axis.'

```

Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Data Structures'
  - 'Arrays'
  - 'Functions'
  - 'NumPy'
CatalogContent:
  - 'learn-python-3'
  - 'paths/data-science'
---

> The **`.amax()`** function returns the maximum value of an array or maximum along an axis. The `.amax()` function is equivalent to the NumPy method `ndarray.max()`.

Reviewer 2: ```suggestion
The **`.amax()`** function returns the maximum value of a given array or the maximum along an axis. The `.amax()` function is equivalent to the NumPy method `ndarray.max()`.

```

## Syntax

```pseudo
numpy.amax(a, axis, out, keepdims, initial, where)
```

The `a` parameter is required and represents the array of elements to choose the maximum from. All other parameters are optional.

Parameters of the `.amax()` function:

- `a`: (Input) The array of elements to choose the maximum from.
- `axis`: (Default = None) An int or a tuple of ints specifying the axis/axes along which to operate. If a tuple of ints, the maximum is across multiple axes.
- `out`: (Default = None) An [`ndarray`](https://www.codecademy.com/resources/docs/numpy/ndarray) to receive result. Must have the same shape as the expected output.
- `keepdims`: (Default = <no value>) A boolean; if `True` will keep reduced axes in the result as dimensions with size one.
- `inital`: (Default = <no value>) The minimum value of an output element.
- `where`: (Default = <no value>) A boolean array that maps to `a` with identical dimensions and selects which elements to compare for the maximum.

> Returns:

Reviewer 2: add an empty line after Returns:

> The maximum element in the array `a`.

Reviewer 2: ```suggestion

```

> - If `axis` is None, the result would be a scalar value.

Reviewer 2: remove the empty lines between these items

- If `axis` is an integer, the result would be an array of dimension `a.ndim - 1`.

- If `axis` is a tuple, the result would be an array of dimension `a.ndim - len(axis)`.

## Example

> The following example creates an array and then uses a few `.amin()` operations to return the minimum.

Reviewer 2: ```suggestion
The following example creates an array and applies several `.amin()` operations, and returns each result to console.

```

```py
import numpy as np

nd = np.array([[2,4,6,8],[10,12,14,16]])

print(np.amax(nd))
print(np.amax(nd, axis=0))
print(np.amax(nd, axis=1))
```

This produces the following output:

```shell
16
[10 12 14 16]
[ 8 16]
```

## Codebyte Example

> The following example creates a 2-dimensional array each containing 8 random integers (between 1 and 20). First, it prints the array, then performs four `.amax()` operations with different arguments to return the maximum values.

Reviewer 2: ```suggestion
The following example creates a 2-dimensional array composed of random integers (between 1 and 20). The code prints the array, and then prints the result of four `.amax()` operations that use different arguments.

```
